clang/lib/DPCT/CallExprRewriterCommon.h

```diff
@@ -792,7 +792,9 @@ inline std::function<std::string(const CallExpr *C)> getDerefedType(size_t Idx)
       DerefQT = ET->getNamedType();
       if (const auto *TDT = dyn_cast<TypedefType>(DerefQT)) {
         auto *TDecl = TDT->getDecl();
-        if (dpct::DpctGlobalInfo::isInCudaPath(TDecl->getLocation()))
+        if (dpct::DpctGlobalInfo::isInCudaPath(TDecl->getLocation()) ||
```

Comment:
How about using `TDT->redecls()` to get all decls.
If one of the decl is in cuda folder, then break.
Is it better?